R/get_uptake_data.r

```diff
@@ -0,0 +1,153 @@
+#' Format uptake data
+#'
+#' Download a local or web .csv file of cumulative uptake percentages.
+#' You must name the columns that contain the state, date, and uptake.
+#'
```

Comment:
Just thinking, is there an API for the data we need? Then we can automatically fetch the data instead of manually downloading! I'm new to getting data from API and would like to know more!